packages/design-system-docs/src/pages/components/FilterTag/_FilterTag.docs.scss

```diff
@@ -0,0 +1,59 @@
+/*
+Filter Tag
+
+Filter Tags are used to display dismissible filter chips
+
+@status Draft
+
+Markup: filtertag.example.html
+
+Style guide: components.filtertag
+*/
+
+/*
+`<FilterTag>`
+
+@react-example FilterTag.example.jsx
+
+@react-props FilterTag.tsx
+
+Style guide: components.filtertag.react
+*/
+
+/*
+---
+
+### When to use
+
+- To display filter results with one or more tags.
+
+### When to consider alternatives
+
+- When a badge is sufficient in place of a filter tag
+
+### Usage
+
+- Do not mix badges and filter tags as it can be confusing to the user
+- Do not use filter tags when a badge is sufficient, ie when none of the tags are removable
+- Filter tags ideally should be short and not wrap to the second line
+
+### Accessibility
+
+- Use `removeText` prop so that screenreaders can accurately read out the close icon action
+- `Enter`, `Backspace`, `Delete`, and `Spacebar` will remove the tag when it is in focus
+
+### Future research
+
+### Customization
+The following Sass variables can be overridden to theme alerts:
+
```

Comment:
I think this empty line needs to be removed so the list shows up - currently none of the variables are showing 
```suggestion

```

Reply:
Got it!